website/docs/configuration/functions/sum.html.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: "functions"
+page_title: "sum - Functions - Configuration Language"
+sidebar_current: "docs-funcs-collection-sum"
+description: |-
+  The sum function takes a list of numbers and returns the sum of those
+  numbers.
+---
+
+# `sum` Function
+
+-> **Note:** This page is about Terraform 0.12 and later. For Terraform 0.11 and
+earlier, see
+[0.11 Configuration Language: Interpolation Syntax](../../configuration-0-11/interpolation.html).
+
+`sum` takes a list of numbers and returns the sum of those numbers.
```

Comment:
I think "list or set of numbers" would be more helpful here (and above in this file).